code/drasil-example/Drasil/Projectile/ProblemDescription.txt

```diff
@@ -0,0 +1,44 @@
+﻿Physical System Description
+
+Elements:
+	1) The object
+	2) The ground
+
+Assumptions
+1) The projectile motion is in 2D.
+2) The object is a point mass.
+3) The object's path starts and ends at ground level.
+4) The effects of drag, wind, and the Earth's curvature are ignored.
+5) The flight is free; there are no collisions during the trajectory of the object.
+6) The positive x-direction is the direction from start to the end of the object's path.
```

Comment:
I don't consider the coordinate system an assumption.  My advice is to put it in the physical system description.  This work starting on a fresh project is a good chance to revisit the rational for separating our knowledge.  I'm sure we aren't 100% consistent in the examples, but the assumptions are intended to define the scope of the problem.  The coordinate system is a modelling decision.  You could have decided to use a polar coordinate system and you would get the same answers (once they are transformed between coordinate systems.)

Reply:
@smiths I based this assumption off of these ones from Chipmunk:

"coordinateSystemTy: The library uses a Cartesian coordinate system."
"axesDefined: The axes are defined using right-handed coordinate system."

Would it be better to say (in the physical system description) "The physical system is modeled in such a way that the positive x-direction is the direction from start to the end of the object's path, and the positive y-direction is the direction away from the ground.", and if so, should I make an issue to update Chipmunk (and potentially others) in a similar fashion?

Reply:
I'm following the example for the requirements for 3dfim+.  We haven't discussed this example previously.  You can find it at:

https://gitlab.cas.mcmaster.ca/SEforSC/se4sc/blob/git-svn/GradStudents/Mojdeh/SRS_3dfim+/3dfim+.pdf

The locations for knowledge in the previous example were guided by the best judgement of myself and the students working on the projects, but we weren't consistent.  The judgement wasn't as guided by principles as it should be, because the principles are not obvious.  This exercise, and the rigorous approach in Drasil, is forcing me to look more explicitly at the assumptions.  However, I'm not 100% sure that the advice I'm giving you will be the same advice I would offer at the end of this exercise.  Let us keep our work to this example, and at the end of the process ask ourselves what principles we applied, and how they should be applied to the other examples.

Reply:
I'll add a Coordinate System section that says "The coordinate system is set up in such a way that the positive x-direction is the direction from start to the end of the object's path and the positive y-direction is the direction away from the ground." Sound good?

Reply:
@smiths Since we've removed ground, would the better alternative be "the positive y-direction is the direction opposite the direction of gravity" for consistency, or "the negative y-direction is the direction of gravity" for brevity?

Reply:
You can just say up and down for the definition of the coordinates and sign convention.  Up is the direction of increasing y coordinate and y is decreasing height.  Gravity is defined after the coordinate directions are assigned.  We could potentially define gravity as acting upward.  If it really is a game, then we can use some strange game physics.  :-)